app/views/statement/accessibility_statement.scala.html

```diff
@@ -84,9 +84,9 @@ <h1>@{AccessibilityStatementMessages.technical_information_header.show}</h1>
         <div>
             <p>@{AccessibilityStatementMessages.technical_information_p1.show}</p>
             <p>@{Html(AccessibilityStatementMessages.technical_information_p2.show)}</p>
-            <h2>@{AccessibilityStatementMessages.technical_information_header2.show}</h2>
+            <h3>@{AccessibilityStatementMessages.technical_information_header2.show}</h3>
```

Comment:
Would it make sense to rename it to: `technical_information_header3` and `technical_information_header4` respectively?